tutorials/scripting/c_sharp/c_sharp_basics.rst

```diff
@@ -112,11 +112,63 @@ In Visual Studio Code:
     for the C# tools plugin to work.
 
 To configure a project for debugging, you need a ``tasks.json`` and ``launch.json`` file in
-the ``.vscode`` folder with the necessary configuration. An example configuration can be
-found `here <https://github.com/godotengine/godot-csharp-vscode/issues/43#issuecomment-1258321229>`__ .
-In the ``launch.json`` file, make sure the ``program`` parameter in the relevant configuration points to your Godot executable, either by
-changing it to the path of the executable or by defining a ``GODOT4`` environment variable that points to the
-executable. Now, when you start the debugger in Visual Studio Code, your Godot project will run.
+the ``.vscode`` folder with the necessary configuration. 
+
+Here is an example ``launch.json``:
+
+.. code-block:: json
+
+    {
+        "version": "0.2.0",
+        "configurations": [
+            {
+                "name": "Play",
+                "type": "coreclr",
+                "request": "launch",
+                "preLaunchTask": "build",
+                "program": "${env:GODOT4}",
+                "args": [],
+                "cwd": "${workspaceFolder}",
+                "stopAtEntry": false,
+            }
+        ]
+    }
+
+For this launch configuration to work, you need to either setup a GODOT4
+environment variable that points to the Godot executable, or replace ``program`` 
+parameter with the path to the Godot executable. On mac or linux, this can be 
+done by adding the following to your .zshrc, .bashrc, or .bash_profile file:
+
+.. code-block:: bash
+    
+    export GODOT4="/Applications/Godot.app/Contents/MacOS/Godot"
```

Comment:
I removed the OS specific mention  here.  It came from the original example.  But I would say that environment variables are the "correct" answer.  Sure a fixed path is the simplest, but it is essentially a hack because it is problematic for source control and working with others or more than one machine.  I would rather at least mention the correct way even if we don't go into much detail.  Especially since it is done differently in the two different IDEs.